source/common/json/config_schemas.cc

```diff
@@ -1042,6 +1042,48 @@ const std::string Json::Schema::GRPC_JSON_TRANSCODER_FILTER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::GZIP_HTTP_FILTER_SCHEMA(R"EOF(
```

Comment:
CONSTRUCT_ON_FIRST_USE

or just declare this as a const char[].

Reply:
Sorry about that 🤦‍♂️

Reply:
@jmarantz Should I change the other ones as well?